bcc/cron-manager-bundle/3.1/config/packages/bcc_cron_manager_bundle.yaml

```diff
@@ -0,0 +1,3 @@
+security:
+    access_control:
+        - { path: ^/admin, roles: ROLE_USER }
```

Comment:
Learning from other packages and discussions: We should not modify other bundles config.

Instead you should put a post-install note that the user need to make this change.

Reply:
I replaced it from the config, post install output will recommendations the user to do it